### PR TITLE
Ubuntu Docker Script for Building PRI

### PR DESCRIPTION
**Pull request by DukeDavis12**
1. The Docker script is able to build the local copy of source code,
     by mounting the local copy into the container and building the mounted local copy.

  2. The script is able to build the latest upstream copy,
     by cloning and building the upstream in /tmp folder of the container and
     finally copying the generated .war and .jar files into the mounted /demo folder of the local copy.

Signed-off-by: Davis Benny <davis.benny@intel.com>

### CODE REVIEW

File: build/build.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+export http_proxy_host=$(echo $http_proxy | awk -F':' {'print $2'} | tr -d '/')
+export http_proxy_port=$(echo $http_proxy | awk -F':' {'print $3'} | tr -d '/')
+
+export https_proxy_host=$(echo $https_proxy | awk -F':' {'print $2'} | tr -d '/')
+export https_proxy_port=$(echo $https_proxy | awk -F':' {'print $3'} | tr -d '/')
+
+export _JAVA_OPTIONS="-Dhttp.proxyHost=$http_proxy_host -Dhttp.proxyPort=$http_proxy_port -Dhttps.proxyHost=$https_proxy_host -Dhttps.proxyPort=$http_proxy_port"
```
Comment (trbehera) [inline build/build.sh]: Typo: -Dhttps_proxyPort=$https_proxy_port

File: build/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:bionic
+
+RUN apt update && \
```
Comment (trbehera) [inline build/Dockerfile]: Replace apt with apt-get

File: build/build.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+export https_proxy_host=$(echo $https_proxy | awk -F':' {'print $2'} | tr -d '/')
+export https_proxy_port=$(echo $https_proxy | awk -F':' {'print $3'} | tr -d '/')
+
+export _JAVA_OPTIONS="-Dhttp.proxyHost=$http_proxy_host -Dhttp.proxyPort=$http_proxy_port -Dhttps.proxyHost=$https_proxy_host -Dhttps.proxyPort=$https_proxy_port"
+
+REMOTE_URL=https://github.com/secure-device-onboard/pri.git
+REMOTE_BRANCH=master
+
+if [ "$use_remote" = "1" ];
```
Comment (trbehera) [inline build/build.sh]: Update indent as per review comments in other PR.

**Review by trbehera — CHANGES_REQUESTED** [review]

File: build/Dockerfile (hunk 2 above)
Comment (DukeDavis12) [reply build/Dockerfile]: Replaced apt with apt-get.

File: build/build.sh (hunk 3 above)
Comment (DukeDavis12) [reply build/build.sh]: Changed indentation to 2-space format.

**Review by trbehera — DISMISSED** [review]

**Comment by trbehera** [comment]
@DukeDavis12 Please rebase and post.

**Review by trbehera — APPROVED** [review]

**Review by prateekcr — APPROVED** [review]

**Pull request merged**